### PR TITLE
Api changes

### PR DESCRIPTION
**Pull request by vishal-shakya-ssir**
Adding POS main branch API changes to DAgent and python backend

### CODE REVIEW

**Review by aswinssir — APPROVED** [review]
Looks Good to me

File: src/pnconnector/src/util/resource.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func ResourcesEventsYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "../resources/events.yaml", size: 39644, mode: os.FileMode(493), modTime: time.Unix(1655975141, 0)}
+	info := bindataFileInfo{name: "../resources/events.yaml", size: 39644, mode: os.FileMode(493), modTime: time.Unix(1660886285, 0)}
```
Comment (mjlee34) [inline src/pnconnector/src/util/resource.go:95]: This change may be included to this pull request becauase resource.go is automatically generated. Does it make sense to include this change?

**Review by next0262 — COMMENTED** [review]
LGTM

**Comment by Seochokid** [comment]
LGTM.

**Pull request merged**